quickstart/src/manifests/overlays/common/kustomization.yaml

```diff
@@ -1,2 +1,4 @@
 bases:
 - ../../bases/nginx/default-ingress
+- ../../bases/tektoncd/base
+- ../../bases/pipeline/base
```

Comment:
TODO: Applying pipeline and tektoncd at the same time, causes an error during bootstrap, because the CRDs do not exist yet.